docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/amazon-ecs.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Running Dagster+ agents on Amazon ECS"
+displayed_sidebar: "dagsterPlus"
+sidebar_position: 11
+sidebar_label: "Amazon ECS"
+---
+
+# Running Dagster+ agents on Amazon ECS
+
+This page provides instructions for running the [Dagster+ agent](dagster-plus/getting-started/whats-dagster-plus#Agents) on an [Amazon ECS](https://aws.amazon.com/ecs/) cluster.
+
```

Comment:
I think the existing prerequisites sections for ECS agents [in a new VPC](https://docs.dagster.io/dagster-plus/deployment/agents/amazon-ecs/creating-ecs-agent-new-vpc#prerequisites) or [an existing VPC](https://docs.dagster.io/dagster-plus/deployment/agents/amazon-ecs/creating-ecs-agent-existing-vpc#prerequisites) are helpful, I would want them in this guide as well